web/impact/impact/graphql/types/expert_profile_type.py

```diff
@@ -132,8 +133,9 @@ def resolve_program_roles(self, info, **kwargs):
         without changing GraphQL queries on the front end.
         """
         user_roles_of_interest = [UserRole.FINALIST, UserRole.ALUM]
```

Comment:
This function is an exact duplicate of the identically-named function on entrepreneur_program_type.py